source/js/components/join/join.jsx

```diff
@@ -490,7 +543,7 @@ export default class JoinUs extends React.Component {
   renderFormContent() {
     if (this.state.apiSuccess) return null;
 
-    let formClass = `d-flex flex-column`;
+    let formClass = `d-flex flex-column join-form`;
```

Comment:
why do we need a new class here?

Reply:
This is due to the changes in nav-newsletter.js, since we’re wrapping `<JoinUs {...props} />` with `<LocalizationProvider>`, we can’t just call `this.form.reset();` directly anymore.

https://github.com/mozilla/foundation.mozilla.org/pull/4181/files#diff-29b89fce3fc1f43c28b636b6a3e9d246R32